### PR TITLE
link inline content update

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

link inline content update

See https://github.com/hashicorp/design-system/pull/934 for similar update to Link::Standalone

### :link: External links

Jira ticket: [HDS-1282](https://hashicorp.atlassian.net/browse/HDS-1282)

### CODE REVIEW

File: website/docs/components/link/inline/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Link::Inline
+title: Inline Link
 description: A link that is used within a body of text.
```
Comment (uxmelina) [inline website/docs/components/link/inline/index.md:3]:
```suggestion
description: A link used within a body of text.
```

File: website/docs/components/link/inline/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 caption: A link that is used within a body of text.
```
Comment (uxmelina) [inline website/docs/components/link/inline/index.md:4]:
```suggestion
caption: A link used within a body of text.
```

**Review by MelSumner — APPROVED** [review]
It looks like the a11y section is missing but I can add this in a subsequent PR.

**Review by KristinLBradley — APPROVED** [review]

**Review by jorytindall — APPROVED** [review]

**Comment by alex-ju** [comment]
Added a commit for the 'How to use' section, which made me realize we probably [don't want to reference the flight website anymore](https://github.com/hashicorp/design-system/pull/967).

**Pull request merged**